parser.y

```diff
@@ -11774,13 +11776,22 @@ PrivType:
 	}
 
 ObjectType:
+	%prec lowerThanFunction
```

Comment:
function is not a reserved keywork in 5.7 (but become reserved after 8.0.1). see https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-8-0-detailed-F

here I didn't change it to reserved to keep compatibility